fee-monitor/src/DynamicChecker.ts

```diff
@@ -73,6 +73,7 @@ function decodeBitsetField(encodedBitSet: number[]): number[] {
 }
 
 export class DynamicChecker {
+    private termClosedRightBefore: boolean = false;
```

Comment:
Instead of creating a new variable, how about using the `parentTermLastBlockNumber`?
`termClosedRightBefore == (parentTermLastBlockNumber + 1 == block.number)`

Reply:
Thank you for your advice. I changed it.